lib/model/model.go

```diff
@@ -1133,14 +1168,17 @@ func (m *model) ClusterConfig(deviceID protocol.DeviceID, cm protocol.ClusterCon
 			fset:         fs,
 			prevSequence: startSequence,
 			evLogger:     m.evLogger,
+			pauseChan:    make(chan struct{}),
```

Comment:
I think we've reached the point where `func newIndexSender(...) *indexSender` should definitely exist